src/main/java/io/github/techstreet/dfscript/screen/script/ScriptEditScreen.java

```diff
@@ -123,7 +126,21 @@ public Rectangle getBounds() {
                     public void render(MatrixStack stack, int mouseX, int mouseY, float tickDelta) {
                         Rectangle b = getBounds();
                         if (b.contains(mouseX, mouseY)) {
-                            DrawableHelper.fill(stack, b.x, b.y, b.x + b.width, b.y + b.height, 0x33000000);
+                            int color = 0x33000000;
+
+                            if(sa.getType().isDeprecated())
```

Comment:
This is bad programming habits in my opinion, it should be inline and line if (sa.getType().isDeprecated()) {